app/views/homepage/_government_activity.html.erb

```diff
@@ -51,9 +51,14 @@
             "track-dimension": "#{t("homepage.index.ministerial_departments_count")} #{t("homepage.index.ministerial_departments")}",
             "track-dimension-index": 29,
             "track-value": 1,
-            "index": 0,
-            "index_total": 2,
-            "section": t("homepage.index.departments_and_organisations")
+            "ga4_link": {
+              "event_name": "navigation",
+              "type": "home page",
+              "index": 0,
+              "index_total": 2,
+              "section": t("homepage.index.departments_and_organisations"),
```

Comment:
We had a thing on some other tracking where it was initially using the translation text but we realised that we'd get different data for the same page in different translations, which was undesirable in that situation. I think in the end we had to hard code the text we needed in the markup (it was the feedback component, I remember now) but a better approach is to call the translation but always force the English version - assuming that's appropriate in this situation.

Reply:
Thank you good spot, have updated